src/java/org/apache/cassandra/service/accord/txn/TxnRead.java

```diff
@@ -51,6 +54,9 @@
 
 public class TxnRead extends AbstractKeySorted<TxnNamedRead> implements Read
 {
+    // There is only potentially one partition in a CAS and SERIAL/LOCAL_SERIAL read
+    public static final String DUMMY_NAME = "";
+    public static final TxnDataName DUMMY = TxnDataName.user(DUMMY_NAME);
```

Comment:
nit/naming: What about something more specific like "SERIAL_READ" (that's also not just an empty string)?

Reply:
Renamed to `SERIAL_READ_NAME`, and `SERIAL_READ` and the value is now `SERIAL_READ`